app/services/induction/amend_participant_cohort.rb

```diff
@@ -148,9 +146,7 @@ def historical_records_changed?
     end
 
     def historical_records
-      return [] unless participant_profile
-
-      @historical_records ||= participant_profile.induction_records.order(created_at: :desc)
+      @historical_records ||= induction_records - [induction_record]
```

Comment:
Is the `#order` not important here?

Reply:
We need no order because the processing of each historical induction record is unique to that record and not depends on anything else.